website/docs/r/logs_pipeline.html.markdown

```diff
@@ -0,0 +1,254 @@
+  * `target` - (Required) Name of the target attribute which value is defined by the matching category.
+  * `category` - (Required) List of filters to match or not a log and their corresponding name to assign a custom value to the log.
+    * `name` - (Required) 
+    * `filter`
+      * `query` - (Required)
+* date_remapper
+  * `name` - (Optional) Name of the processor.
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* grok_parser
+  * `name` - (Optional) Name of the processor.
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `source` - (Required) Name of the log attribute to parse.
+  * `grok`
+    * `support_rules` - (Required) Support rules for your grok parser.
+    * `match_rules` - (Required) Match rules for your grok parser.
+* message_remapper
+  * `name` - (Optional) Name of the processor.
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* pipeline
+  * `name` - (Optional) Name of the nested pipeline.
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `filter` - (Required) Defines the nested pipeline filter. Only logs that match the filter criteria are processed by this pipeline.
+    * `query` - (Required)
+  * `processor` - (Optional) [Processors](logs_pipeline.html#Processors). It can not be pipelines anymore.
+* service_remapper
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* status_remapper
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* trace_id_remapper
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* url_parser
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes. 
+  * `target` - (Required) Name of the parent attribute that contains all the extracted details from the `sources`.
+  * `normalize_ending_slashes` - (Optional, default = false) If needs to normalize the ending slashes or not.
+* user_agent_parser
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+  * `target` - (Required) Name of the parent attribute that contains all the extracted details from the sources.
+  * `is_encoded` - (Optional, default = false) If the source attribute is url encoded or not.
+
+## Attributes Reference
```

Comment:
I'd put this argument reference for pipelineorder in a separate doc just for that resource type